src/modules/common/components/bars/topBar/topBar.js

```diff
@@ -16,7 +16,7 @@ import styles from './topBar.css';
 import Network from './networkName';
 import NavigationButtons from './navigationButtons';
 
-const TopBar = ({ t, history, network, token, noOfVotes, location }) => {
+const TopBar = ({ t, history, noOfVotes, location }) => {
```

Comment:
you can remove t from prop and consume it like this

`const { t } = useTranslation();`